packages/frontend-api/src/Model/Complaint/ComplaintItemDataApiFactory.php

```diff
@@ -31,8 +34,13 @@ public function __construct(
      */
     public function createFromComplaintItemInput(OrderItem $orderItem, array $item): ComplaintItemData
     {
+        $product = $orderItem->getProduct();
+
         $complaintItemData = $this->complaintItemDataFactory->create();
         $complaintItemData->orderItem = $orderItem;
+        $complaintItemData->product = $product;
+        $complaintItemData->productName = $product->getName($this->domain->getLocale());
```

Comment:
this would work the same even without `$this->domain->getLocale()` or am I missing something?

Reply:
btw, the product might be null, so probably `$orderItem->getName()` should be used here

Reply:
fixed [fixup! complaints b2b customers support](https://github.com/shopsys/shopsys/pull/3295/commits/585c5b09e615c10c4351fc201d7abf686fa2e022)